_docs/payout-details.md

```diff
@@ -553,31 +555,93 @@ For South African bank payments please use the following recipient details:
+Government Owned Entity Trusts: government_owned_entity
+GO (Majority Owned Subsidiary of State-Owned Company): go
+Financial Institution: financial_institution
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="zar-entity-types" raw=data-raw %}
+
+The valid values for `nature_of_business` are the following:
 
 {% capture data-raw %}
 ```
-Standard Bank: 051001
-First National Bank: 250655
-ABSA: 632005
-Nedbank: 198765
-Investec: 580105
-Capitec Bank: 470010
-Bank of Athens: 410506
-Bidvest Bank: 462005
-African Bank: 430000
-Mercantile Bank: 450905
-SA Post Office: 460005
-Tyme Bank: 678910
-Ubank: 431010
-Discovery Bank: 679000
-Bank Zero: 888000
+- personal: Personal
+- agriculture_and_hunting: Agriculture and Hunting
+- forestry: Forestry
+- fishing: Fishing
+- agricultural_by_products: Agricultural By-Products
+- coal_mining: Coal Mining
+- oil_mining: Oil Mining
+- iron_ore_mining: Iron Ore Mining
+- other_metal_and_diamond_mining: Other Metal and Diamond Mining
+- other_mineral_mining: Other Mineral Mining
+- manufacturing_of_food_drink_tobacco: Manufacture of Food/Drink/Tobacco
+- manufacturing_of_textiles_leather_fur_furniture: Manufacture of Textiles/Leather/Fur/Furniture
+- manufacture_of_wooden_products_furniture: Manufacture of Wooden Products/Furniture
+- manufacture_of_paper_pulp_allied_products: Manufacture of Paper/Pulp/Allied Products
+- manufacture_of_chemicals_medical_petroleum_rubber_plastic_products: Manufacture Of Chemicals Medical Petroleum Rubber Plastic Products
+- manufacture_of_pottery_china_glass_stone: Manufacture Of Pottery China Glass Stone
+- manufacture_of_iron_steel_non_ferrous_metals_basic_industries: Manufacture Of Iron Steel Non-Ferrous Metals Basic Industries
+- manufacture_of_metal_products_electrical_and_scientific_engineering: Manufacture Of Metal Products Electrical And Scientific Engineering
+- manufacture_of_jewelry_musical_instruments_toys: Manufacture Of Jewelry Musical Instruments Toys
+- electricity_gas_and_water: Electricity, Gas And Water
+- construction: Construction
+- wholesale_trade: Wholesale Trade
+- retail_trade: Retail Trade
+- catering_incl_hotels: Catering Incl. Hotels
+- transport_storage: Transport Storage
+- communications: Communications
+- finance_and_holding_companies: Finance And Holding Companies
+- insurance: Insurance
+- business_services: Business Services
+- real_estate_development_investment: Real Estate Development Investment
+- central_state_governments: Central State Governments
+- community_services_defence_police_prisons_etc: Community Services Defence Police Prisons Etc
+- social_services_education_health_care: Social Services Education Health Care
+- personal_services_leisure_services: Personal Services - Leisure Services
+- personal_services_domestic_laundry_repairs: Personal Services - Domestic Laundry Repairs
+- personal_services_embassies_international_organisations: Personal Services - Embassies
```

Comment:
would be good to unify key/value order in both of these enum sets

Reply:
@mchochowski change made, also updated the business sender nature_of_business so they're aligned